specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2024-01-01-preview/examples/recommendations/PatchRecommendation.json

```diff
@@ -18,38 +18,27 @@
       },
       "body": {
```

Comment:
do you think we should change the body to reflect that only the status can be updated?

Reply:
i.e. by of type RecommendationPatch

Reply:
I'm not sure I understand, are you talking about the response body?
Or the request?
In `RecommendationPatch` we only have the `state` field as updatable.

![image](https://github.com/Azure/azure-rest-api-specs/assets/19761142/d2b1778b-8412-4cab-873f-079504d3439b)

Reply:
We do return the whole recommendation

![image](https://github.com/Azure/azure-rest-api-specs/assets/19761142/891dada2-11d0-491a-8383-0e93e986097f)

Reply:
RecommendationPatch